MP campus circuit and dorm start

phbr1768 opened the pull request:
Fixed the scene path in multiplayer for campus circuit and dorm
Fixed the time scale for client pause in multiplayer

LRLogan reviewed (changes requested):
Races don't end after you finish. Otherwise it looks like the problem this intended to fix has been fixed as well as the API still sending back the correct data. Although I had to use the unity build to check this. In person people should try loading up these maps and make sure the correct data is sent back.

phbr1768 commented:
Fixed the scene path in multiplayer for campus circuit and dorm
 - the assignments of scene loading in the MultiplayerSceneLoader.cs have been updated to match the string assignments in GameManager.cs
Fixed the time scale for client pause in multiplayer
- clients don't set the time scale locally anymore

WhispDoor reviewed (approved):
This solves more problems than it causes. Multiplayer is real wonky generally, but its gotta be added.

phbr1768 commented:
> This solves more problems than it causes. Multiplayer is real wonky generally, but its gotta be added.

it doesnt cause any problems >:(
